cli/command/image/build.go

```diff
@@ -235,6 +247,28 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 		}
 	}
 
+	// replace Dockerfile if added dynamically
```

Comment:
I think this comment needs to be updated based on the recent change you made to generate a random dockerfile name.